bench/cmd/main.go

```diff
@@ -4,18 +4,35 @@ import (
 	"fmt"
 	"github.com/kyokan/plasma/bench"
 	"os"
+        "runtime"
+        "runtime/trace"
+        "time"
 )
 
 func main() {
+        runtime.GOMAXPROCS(16)
```

Comment:
going to remove `runtime.GOMAXPROCS(16)`, it defaults to the # of CPUS, so won't make a material difference